### PR TITLE
Rossbranch

### PR DESCRIPTION
**Pull request by rdtauche**

### CODE REVIEW

**Review by csj5118 — APPROVED** [review]
LGTM

**Review by csj5118 — APPROVED** [review]
LGTM

File: client/src/components/Footer.jsx
Comment (Tylerz3936) [inline client/src/components/Footer.jsx:1]: looks fine

**Review by csj5118 — APPROVED** [review]
LGTM

**Review by csj5118 — APPROVED** [review]
LGTM

**Pull request merged**